src/signals/incident/containers/KtoContainer/components/KtoForm/index.test.js

```diff
@@ -90,38 +33,67 @@ describe('<KtoForm />', () => {
     spy = jest.spyOn(instance, 'setValues');
   });
 
-  afterEach(() => {
-    jest.runAllTimers();
-    jest.resetAllMocks();
-  });
-
   describe('rendering', () => {
     it('expect to render YES form correctly', () => {
-      wrapper.setProps({
+      const { queryByText, rerender } = render(
+        withAppContext(
+          <KtoForm {...props} />,
+        ),
+      );
+
+      const yesProps = {
+        ...props,
         ktoContainer: {
           form: {
+            is_satisfied: true,
             yesNo: 'ja',
           },
           answers: {
             'Antwoord JA': 'Antwoord JA',
           },
         },
-      });
-      expect(wrapper).toMatchSnapshot();
+      };
+
+      rerender(
+        withAppContext(
+          <KtoForm {...yesProps} />,
+        ),
+      );
+
+      expect(queryByText('Waarom bent u tevreden?')).toBeInTheDocument();
```

Comment:
Can you refer to the props in the fixture instead of hardcoding these texts? Makes the assertions a bit more flexible.

Reply:
gedaan